### PR TITLE
Add disk status to diagnostic report and fix line break.

### LINKED ISSUE #721: Add Disk write status to diagnostic report
Blockchain team has added the ability to detect whether a disk (or SD card) is read-only, or is ok (able to write). 
Depends on https://github.com/helium/gateway-config/pull/73

Read-only disks means sync is stalled, unable to poc, and needs to be replaced.
OK means everything is fine.

We'll add a new line item to diagnostic report to show this:
Disk    x Read-Only 

or

Disk    OK

If read-only, add additional text:
Contact your Manufacturer for a replacement. Hotspot unable to sync due to hardware failure.

image

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Closes #721

### CODE REVIEW

File: src/features/hotspots/settings/HotspotDiagnosticReport.tsx [hunk 1]
```diff
@@ -122,6 +131,16 @@ const HotspotDiagnosticReport = ({ onFinished }: Props) => {
           ? formatMac(diagnostics.eth)
           : t('generic.unavailable'),
       },
+      {
+        attribute: t('hotspot_settings.diagnostics.disk'),
+        value: diagnostics?.disk
+          ? formatMac(diagnostics.disk)
```
Comment (tyler-whitman) [inline src/features/hotspots/settings/HotspotDiagnosticReport.tsx:137]: Are the values of this `"read only"` and `"disk ok"`? Will formatMac work for those?

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Mine shows Unavailable, but it hasn't got the firmware update yet. One other question for you.

File: src/features/hotspots/settings/HotspotDiagnosticReport.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/hotspots/settings/HotspotDiagnosticReport.tsx]: I fixed this up based on the github issue. Good catch!

**Comment by matthewcarlreetz** [comment]
> Mine shows Unavailable, but it hasn't got the firmware update yet. One other question for you.

Yeah, the hotspot update will go out with the next GA release.

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**